giftcard/src/main/java/com/adyen/checkout/giftcard/internal/ui/model/GiftCardOutputData.kt

```diff
@@ -8,10 +8,12 @@
 
 package com.adyen.checkout.giftcard.internal.ui.model
 
+import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.core.internal.ui.model.FieldState
 import com.adyen.checkout.components.core.internal.ui.model.OutputData
 
-internal data class GiftCardOutputData(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class GiftCardOutputData(
```

Comment:
Same here

Reply:
Since `GiftCardDelegate` has a reference to it we cannot have this as `internal`.